apps/omg_api/test/block_queue/core_test.exs

```diff
@@ -571,5 +577,21 @@ defmodule OMG.API.BlockQueue.CoreTest do
                assert {:error, :account_locked} = process_submit_result(submission, @account_locked_response, 0)
              end) =~ "[error]"
     end
+
+    @tag fixtures: [:empty_with_gas_params]
+    test "gas price change only, when try to push blocks", %{empty_with_gas_params: state} do
```

Comment:
I'm not digging into this PR, but please double check if theres a test that `gas price changes only, when ethereum advances`. Also the test case mentioned before `gas price doesn't change when ethereum backs off, even if block in queue`.

Are there also appropriate tests for when to lower the price, according to our revised behaviors we've talked about?

Also make sure that the documentation in `moduledocs` are up to date and reflect the behaviors we're implementing here - when price is lowered, when it is raised.